db/migrate/20211101075313_change_assest_to_asset.rb

```diff
@@ -0,0 +1,5 @@
+class ChangeAssestToAsset < ActiveRecord::Migration[6.1]
+  def change
+    rename_table :assests, :assets
```

Comment:
you could have made these changes in the `create_assets` migration by rolling it back. as it was not already committed/merged to main branch

Reply:
Yes, I may have.
I think I did not do it because I found the error much later when a lot of work was done.
I'll keep that in mind the next time.